src/Evolution/Actions/ComputeVolumeFluxes.hpp

```diff
@@ -50,6 +51,42 @@ struct ComputeVolumeFluxes {
       const ParallelComponent* const /*meta*/) noexcept {  // NOLINT const
     db::mutate_apply<typename Metavariables::system::volume_fluxes>(
         make_not_null(&box));
+    if (static_cast<bool>(
+            db::get<domain::Tags::MeshVelocity<Metavariables::volume_dim>>(
+                box))) {
+      using variables_tag = typename Metavariables::system::variables_tag;
+      using fluxes_tag =
+          db::add_tag_prefix<::Tags::Flux, variables_tag,
+                             tmpl::size_t<Metavariables::volume_dim>,
+                             Frame::Inertial>;
+      db::mutate<fluxes_tag>(
+          make_not_null(&box),
+          [](const auto fluxes_ptr, const auto& evolved_vars,
+             const tnsr::I<DataVector, Metavariables::volume_dim,
+                           Frame::Inertial>& grid_velocity) noexcept {
+            tmpl::for_each<
+                typename Metavariables::system::volume_fluxes::return_tags>(
+                [&evolved_vars, &fluxes_ptr,
+                 &grid_velocity](auto tag_v) noexcept {
+                  using ::get;
+                  using flux_tag = typename decltype(tag_v)::type;
+                  using vars_tag = db::remove_tag_prefix<flux_tag>;
+                  auto& flux_tensor = get<flux_tag>(*fluxes_ptr);
+                  for (size_t i = 0; i < flux_tensor.size(); ++i) {
+                    const auto flux_tensor_index =
+                        flux_tensor.get_tensor_index(i);
+                    const auto tensor_index =
+                        all_but_specified_element_of(flux_tensor_index, 0);
+
+                    flux_tensor[i] -=
+                        get<vars_tag>(evolved_vars).get(tensor_index) *
+                        grid_velocity.get(gsl::at(flux_tensor_index, 0));
+                  }
+                });
+          },
+          db::get<variables_tag>(box),
+          *db::get<domain::Tags::MeshVelocity<Metavariables::volume_dim>>(box));
```

Comment:
Please modify the Doxygen documentation (up above where I can't attach a comment) of this action to
- update the list of things used from the DataBox (as far as I can tell, need to add MeshVelocity)
- update the description to point out that adds the moving mesh term if moving mesh is on. Would be helpful to either give the formula, or point to a "centralized" moving mesh documentation where the formula can be found. (can do this part later if this is still coming in a future PR)

Reply:
1. I'll update the docs, thanks!
2. I can add the formula here. We currently don't have any central DG documentation and I don't know of a paper (ye0) that actually does DG correctly in curved spacetimes, on moving meshes, and with this formulation. I think we should document how we do DG, probably more than even just pointing to a paper. That should probably be a `tutorial`. What do you think?

Reply:
A tutorial would definitely be nice. Until we have that and/or a paper, I think giving the formula in the dox (just so users don't have to figure it out by reading source code) is the way to go 👍